skimage/restoration/deconvolution.py

```diff
@@ -326,27 +329,55 @@ def unsupervised_wiener(image, psf, reg=None, user_params=None, is_real=True,
     return (x_postmean, {'noise': gn_chain, 'prior': gx_chain})
 
 
-def richardson_lucy(image, psf, iterations=50, clip=True):
-    """Richardson-Lucy deconvolution.
-
+def _pad_to_shape(psf, image):
```

Comment:
Would it be possible to swap these two arguments (to keep image first), which is more conventional and more coherent with the rest.

Reply:
fix it